src/fiatconnect/KycLanding.tsx

```diff
@@ -67,9 +76,10 @@ const useComponentSize = (): [
   { width: number; height: number },
   (event: LayoutChangeEvent) => void
 ] => {
+  const { width, height } = Dimensions.get('window')
```

Comment:
before there would be a moment where the "Grey'd out" portion was 0 width/height so it would appear not grey'd out at all. Now the first paint is grey'd out